xbmc/cores/AudioEngine/Sinks/AESinkDARWINIOS.mm

```diff
@@ -11,574 +11,585 @@
 #include "cores/AudioEngine/Utils/AEUtil.h"
 #include "cores/AudioEngine/Utils/AERingBuffer.h"
 #include "cores/AudioEngine/Sinks/darwin/CoreAudioHelpers.h"
-#include "ServiceBroker.h"
-#include "platform/darwin/DarwinUtils.h"
```

Comment:
make a separate PR for the AESinkDARWINIOS change as it affects iOS too

Reply:
@kambala-decapitator Do you want to open this PR?

Reply:
sure, will do

Reply:
Thank you ;-)
May I restore to master version for this file on this PR?

Reply:
I guess so, just keep in mind that tvOS project won't compile with it.

Reply:
Yes, but I assume that your future PR will be merged before this one. So a rebase onto master once your PR will be merged should resolve the issue ;-)

Reply:
I revert this file to master state. Wait until kambala's PR created and merged for this file modification.